vulcan/lib/client/jsx/components/vulcan_nav.jsx

```diff
@@ -116,8 +117,12 @@ const Halo = ({radius}) => {
 };
 
 function Logo() {
-  let {state} = useContext(VulcanContext);
-  const {calculating} = state;
+  const {state} = useContext(VulcanContext);
+  const {workflow, status} = state;
+
+  let calculating = workflow ? workflow.steps[0].some(
+    step => statusStringOfStepOrGroupedStep(step, workflow, status) == 'running'
```

Comment:
Just curious why you used `statusStringOfStepOrGroupedStep` instead of just `statusOfStep`? If you're pulling steps directly from the workflow, I don't think any will be Grouped Steps...?

Reply:
It still works either way.  The function behaves the same.  I think using this function is fine.

Reply:
It seems statusString... does some additional work: `return statusOfStep(step, status)?.status || STATUS.PENDING; ` - I tried using statusOfStep at first but duplicating this logic in the throbber seems like a bad idea.

Reply:
Ah, okay! :+1: